### ISSUE TITLE
weird EP distribution in QSHMM-ONT-HQ.model

### ISSUE THREAD

**Issue by ocxtal** [yukiteruono/pbsim3#21]
When I used the latest pbsim3 (3.0.2) with the `--qshmm data/QSHMM-ONT-HQ.model` option, I found some of generated reads have strange quality strings where both of sequence and quality look sequence:

![Screenshot 2024-02-25 at 13 36 52](https://github.com/yukiteruono/pbsim3/assets/16234240/32542deb-51f6-41e8-9583-a5697bd16ec7)

I investigated the code and the model file, and found columns correspond to `A`, `C`, `G`, and `T` for `97 EP` lines in `QSHMM-ONT-HQ.model` have much higher values than the other columns of `97 EP`. Is this due to a bug in the training script?

**Comment by yukiteruono** [comment]
Thank you for reporting the trouble.
Due to a bug, most of the training data (quality score) for accuracy=97 was replaced with nucleotide sequences.
It will take about a week to correct the model. Temporarily removed QSHMM-ONT-HQ.model and ERRHMM-ONT-HQ.model.

**Comment by yukiteruono** [comment]
We have confirmed that there are no problems with models other than QSHMM-ONT-HQ.model.

**Comment by ocxtal** [comment]
Thank you for investigation. For my own evaluation purpose, I've replaced the entire values of `97 EP` with the `96 EP` ones, and the generated sequences look fine.

**Comment by yukiteruono** [comment]
Please use v3.0.4.

**Comment by ocxtal** [comment]
Thanks, I'll try it.
